views/InfoboxView.vue

```diff
@@ -28,56 +28,58 @@ const props = defineProps( {
 </script>
 
 <template>
-  <UCard class="flex flex-col items-center justify-center">
-    <template #header>
-      <div class="flex flex-col items-center justify-center">
-        <img
-            :src="imageUrl"
-            alt="image"
-            class="w-40 object-cover pb-2"
-            :class="{
-            'blur-none': blur === 0 || over,
-            'blur-sm': blur === 1,
-            'blur': blur === 2,
-            'blur-md': blur === 3,
-            'blur-lg': blur === 4,
-            'blur-xl': blur === 5,
-            'blur-2xl': blur === 6,
-            'blur-3xl': blur === 7
-          }"
-        />
+  <div class="pt-3">
+    <UCard class="flex flex-col items-center justify-center pt-3">
+      <template #header>
+        <div class="flex flex-col items-center justify-center">
+          <img
+              :src="imageUrl"
+              alt="image"
+              class="w-40 object-cover pb-2"
+              :class="{
+              'blur-none': blur === 0 || over,
+              'blur-sm': blur === 1,
+              'blur': blur === 2,
+              'blur-md': blur === 3,
+              'blur-lg': blur === 4,
+              'blur-xl': blur === 5,
+              'blur-2xl': blur === 6,
+              'blur-3xl': blur === 7
+            }"
+          />
```

Comment:
```suggestion
          <img :src="imageUrl"  alt="image" class="w-40 object-cover pb-2" :class="{
              'blur-none': blur === 0 || over,
              'blur-sm': blur === 1,
              'blur': blur === 2,
              'blur-md': blur === 3,
              'blur-lg': blur === 4,
              'blur-xl': blur === 5,
              'blur-2xl': blur === 6,
              'blur-3xl': blur === 7
            }"
          />
```
Again, you can save several lines.